fe/fe-core/src/main/java/com/starrocks/metric/MetricRepo.java

```diff
@@ -507,6 +513,181 @@ public Long getValue() {
         STARROCKS_METRIC_REGISTER.addMetric(tpcOutSegs);
     }
 
+    public static void initMemoryMetrics() {
+        GaugeMetric<Long> tabletCnt = new GaugeMetric<Long>("memory", MetricUnit.NOUNIT,
+                "The count of tablets") {
+            @Override
+            public Long getValue() {
+                return GlobalStateMgr.getCurrentInvertedIndex().getTabletCount();
+            }
+        };
+        tabletCnt.addLabel(new MetricLabel("type", "tablet_count"));
+        STARROCKS_METRIC_REGISTER.addMetric(tabletCnt);
+
+        GaugeMetric<Long> tabletBytes = new GaugeMetric<Long>("memory", MetricUnit.BYTES,
+                "The bytes of tablets") {
+            @Override
+            public Long getValue() {
+                return GlobalStateMgr.getCurrentInvertedIndex().getTabletCount()
+                        * SizeEstimator.estimate(new LocalTablet());
```

Comment:
will this count the ref to `private List<Replica> replicas;` member?

Reply:
Yes, but for the `new LocalTablet()`, the repicas is null, so only the basic types will be included.